From 4413af207fe1000bd8983404c09aaf2bc58bee57 Mon Sep 17 00:00:00 2001
From: Raphael Mounier <mounierr07@gmail.com>
Date: Fri, 7 Nov 2025 17:32:01 +0100
Subject: [PATCH] Remove empty include file code_coverage.arm policy

Empty file cause compilation failed and crash after omx hal on vndk28

Change-Id: I21c9362cc710c3b07877c3cbed0b252e84b819ed
---
 services/mediacodec/seccomp_policy/mediacodec-arm.policy   | 2 +-
 services/mediacodec/seccomp_policy/mediacodec-arm64.policy | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/services/mediacodec/seccomp_policy/mediacodec-arm.policy b/services/mediacodec/seccomp_policy/mediacodec-arm.policy
index b4a9ff6249..14cf31821a 100644
--- a/services/mediacodec/seccomp_policy/mediacodec-arm.policy
+++ b/services/mediacodec/seccomp_policy/mediacodec-arm.policy
@@ -59,5 +59,5 @@ getdents64: 1
 getrandom: 1
 
 @include /system/etc/seccomp_policy/crash_dump.arm.policy
+# @include /system/etc/seccomp_policy/code_coverage.arm.policy
 
-@include /system/etc/seccomp_policy/code_coverage.arm.policy
diff --git a/services/mediacodec/seccomp_policy/mediacodec-arm64.policy b/services/mediacodec/seccomp_policy/mediacodec-arm64.policy
index b4a9ff6249..14cf31821a 100644
--- a/services/mediacodec/seccomp_policy/mediacodec-arm64.policy
+++ b/services/mediacodec/seccomp_policy/mediacodec-arm64.policy
@@ -59,5 +59,5 @@ getdents64: 1
 getrandom: 1
 
 @include /system/etc/seccomp_policy/crash_dump.arm.policy
+# @include /system/etc/seccomp_policy/code_coverage.arm.policy
 
-@include /system/etc/seccomp_policy/code_coverage.arm.policy
-- 
2.25.1

